qm-plugins/qm-cron/class-qm-cron-output-html.php

```diff
@@ -0,0 +1,333 @@
+		}
+
+		return $attributes;
+	}
+
+	/**
+	* Display the timing for the event as a date, timestamp and human readable time difference.
+	*
+	* @param int $time        Timestamp.
+	* @param int $event_count Number of events running at this time.
+	*/
+	private function display_event_time( $time, $event_count ) {
+		$row_span = ( $event_count > 1 ) ? ' rowspan="' . esc_attr( $event_count ) . '"' : '';
+
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+		echo '<td' . $row_span . '>
+		   ', esc_html( gmdate( 'Y-m-d H:i:s', $time ) ), '<br />
+		   ', esc_html( $time ), '<br />
+		   ', '<i>' . esc_html( $this->display_past_time( human_time_diff( $time ), $time ) ) . '</i>', '
+	   </td>';
+	}
+
+
+	/**
+	* Display the name of the cron job event hook.
+	*
+	* @param string $hook          Hook name.
+	* @param int    $arg_set_count Number of events running at this time and on this hook.
+	*/
+	private function display_event_hook( $hook, $arg_set_count ) {
+		$row_span = ( $arg_set_count > 1 ) ? ' rowspan="' . esc_attr( $arg_set_count ) . '"' : '';
+
+		echo '<td' . $row_span . '>', esc_html( $hook ), '</td>'; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	}
+
+
+	/**
+	* Displays the the event schedule name for recurring events or else 'single event'.
+	*
+	* @param array $info Event info array.
+	*/
+	private function display_event_schedule( $info ) {
+		if ( ! empty( $info['schedule'] ) ) {
+			echo esc_html( $info['schedule'] );
+		} else {
+			esc_html_e( 'Single Event', 'query-monitor' );
+		}
+	}
+
+
+	/**
+	* Displays the event interval in seconds, minutes and hours.
+	*
+	* @param array $info Event info array.
+	*/
+	private function display_event_intervals( $info ) {
+		if ( ! empty( $info['interval'] ) ) {
+			$interval = (int) $info['interval'];
+			/* translators: %s is number of seconds. */
+			printf( esc_html__( '%ss', 'query-monitor' ) . '<br />', esc_html( $interval ) );
+			/* translators: %s is number of minutes. */
+			printf( esc_html__( '%sm', 'query-monitor' ) . '<br />', esc_html( $this->get_minutes( $interval ) ) );
+			/* translators: %s is number of hours. */
+			printf( esc_html__( '%sh', 'query-monitor' ), esc_html( $this->get_hours( $interval ) ) );
+			unset( $interval );
+		} else {
+			esc_html_e( 'Single Event', 'query-monitor' );
+		}
+	}
+
+	/**
+	* Displays the cron arguments in a readable format.
+	*
+	* @param mixed $args Cron argument(s).
+	*
+	* @return void
+	*/
+	private function display_event_cron_arguments( $args ) {
+		// Arguments defaults to an empty array if no arguments are given.
+		if ( is_array( $args ) && array() === $args ) {
```

Comment:
```suggestion
		if ( is_array( $args ) && empty( $args ) ) {
```

?

(so that our code looks the same [everywhere](https://github.com/Automattic/vip-go-mu-plugins/pull/3832/files#diff-7f540689ad3ddbab9b7839e5f2bdc5e2ac83869f7c7aecbb64fc94e17b6120eaR56))